nornir_shared/plot.py

```diff
@@ -224,32 +224,31 @@ def PolyLine(PolyLineList, Title=None, XAxisLabel=None, YAxisLabel=None, OutputF
         ...
     '''
 
-    colors = ['black', 'blue', 'green', 'yellow', 'orange', 'red', 'purple']
     # print Hist.Bins
     plt.cla()
+    num = 0
     for line in PolyLineList:
-        numPoints = len(line[0])
-        colorVal = 'black'
-        
-        if  numPoints < len(colors):
-            colorVal = colors[numPoints]
-            
+        color = 'red'
+        if len(colors) > num:
+            color = colors[num]
+            print(color)
```

Comment:
print needs to be removed.  This function is called a lot when generating reports and each call to print will take a lock and create debug spew.